### PR TITLE
fix: Close jump function of clicking pod in PVC detail page of multi-cluster

### PR DESCRIPTION
**Pull request by weili520**
Signed-off-by: TheYoungManLi <1512298456@qq.com>

### What type of PR is this?
/kind bug

### Which issue(s) this PR fixes:
Fixes ##2175

### Special notes for reviewers:
截屏2021-09-06 16 23 54

### Does this PR introduced a user-facing change?
```release-note
NONE
```

### Additional documentation, usage docs, etc.:
```docs

```

### CODE REVIEW

File: src/pages/fedprojects/containers/Volumes/Detail/VolumeMounts/PodsCard/index.jsx [hunk 1]
```diff
@@ -51,7 +51,7 @@ export default class PodsCardWrapper extends Component {
         <div className={styles.content}>
           <PodsCard
             detail={detail}
-            prefix={this.prefix}
+            // prefix={this.prefix}
```
Comment (xuliwenwenwen) [inline src/pages/fedprojects/containers/Volumes/Detail/VolumeMounts/PodsCard/index.jsx:54]: Dead code can be deleted.

Comment (weili520) [reply src/pages/fedprojects/containers/Volumes/Detail/VolumeMounts/PodsCard/index.jsx]: done, thanks. :)

**Pull request merged**